hack/images/kicbase.Dockerfile

```diff
@@ -63,12 +65,12 @@ USER root
 RUN mkdir -p /kind
 # Deleting leftovers
 RUN apt-get clean -y && rm -rf \
-  /var/cache/debconf/* \
-  /var/lib/apt/lists/* \
-  /var/log/* \
-  /tmp/* \
-  /var/tmp/* \
-  /usr/share/doc/* \
-  /usr/share/man/* \
-  /usr/share/local/* \
-  RUN echo "kic! Build: ${COMMIT_SHA} Time :$(date)" > "/kic.txt"
+    /var/cache/debconf/* \
+    /var/lib/apt/lists/* \
+    /var/log/* \
+    /tmp/* \
+    /var/tmp/* \
+    /usr/share/doc/* \
+    /usr/share/man/* \
+    /usr/share/local/* \
+    RUN echo "kic! Build: ${COMMIT_SHA} Time :$(date)" > "/kic.txt"
```

Comment:
while we are at this please un-indent this RUN, this was a mistake indendation.

Reply:
Done.